ui/app/services/permissions.js

```diff
@@ -0,0 +1,36 @@
+import Service, { inject as service } from '@ember/service';
+import { task } from 'ember-concurrency';
+
+// decide what happens when no policies are found
+// because requesting the root token's ACL policy always returns nil
+// https://github.com/hashicorp/vault/pull/4386/files#diff-a145099c8e3d917858619dcfb2ae09b9R3464
```

Comment:
This returns `root: true` in the data block of the response - `nil` in that go code is the error.